python/qpid_dispatch/management/qdrouter.json

```diff
@@ -841,6 +841,46 @@
                     "type": "string",
                     "description": "Where to send log messages. Can be 'stderr', 'stdout', 'syslog' or a file name.",
                     "update": true
+                },
+                "default": {
```

Comment:
These attributes should have names that are more descriptive and should suggest that they are counters.